New CRAN warning: identifier preceded by whitespace in a literal operator declaration

Hi All,

CRAN has started flagging a new significant warning in the cpp11 headers:

```
Found the following significant warnings:
  ../inst/include/cpp11/R.hpp:54:31: warning: identifier '_xl' preceded by whitespace in a literal operator declaration is deprecated [-Wdeprecated-literal-operator]
  ../inst/include/cpp11/named_arg.hpp:44:29: warning: identifier '_nm' preceded by whitespace in a literal operator declaration is deprecated [-Wdeprecated-literal-operator]
```

Here's the relevant [checking and install log](https://www.r-project.org/nosvn/R.check/r-devel-linux-x86_64-fedora-clang/QuickJSR-00check.html)

The fix is simple enough (changing the declaration from `operator"" _xl` to `operator""_xl`), but just flagging here for awareness

Thanks!

--- a/NEWS.md
+++ b/NEWS.md
@@ -1,5 +1,7 @@
 # cpp11 (development version)
 
+* Fixed an issue related to `-Wdeprecated-literal-operator` (#447, @andrjohns).
+
 # cpp11 0.5.1
 
 * cpp11 now requires R >=4.0.0, in line with the

--- a/inst/include/cpp11/R.hpp
+++ b/inst/include/cpp11/R.hpp
@@ -49,7 +49,7 @@
 namespace cpp11 {
 namespace literals {
 
-constexpr R_xlen_t operator"" _xl(unsigned long long int value) { return value; }
+constexpr R_xlen_t operator""_xl(unsigned long long int value) { return value; }
 
 }  // namespace literals
 

--- a/inst/include/cpp11/named_arg.hpp
+++ b/inst/include/cpp11/named_arg.hpp
@@ -39,7 +39,7 @@ class named_arg {
 
 namespace literals {
 
-inline named_arg operator"" _nm(const char* name, std::size_t) { return named_arg(name); }
+inline named_arg operator""_nm(const char* name, std::size_t) { return named_arg(name); }
 
 }  // namespace literals
 

--- a/inst/include/fmt/format.h
+++ b/inst/include/fmt/format.h
@@ -2749,7 +2749,7 @@ constexpr auto operator""_a()
   return {};
 }
 #else
-constexpr auto operator"" _a(const char* s, size_t) -> detail::udl_arg<char> {
+constexpr auto operator""_a(const char* s, size_t) -> detail::udl_arg<char> {
   return {s};
 }
 #endif
@@ -2764,7 +2764,7 @@ constexpr auto operator"" _a(const char* s, size_t) -> detail::udl_arg<char> {
     std::string message = "The answer is {}"_format(42);
   \endrst
  */
-constexpr auto operator"" _format(const char* s, size_t n)
+constexpr auto operator""_format(const char* s, size_t n)
     -> detail::udl_formatter<char> {
   return {{s, n}};
 }

